escalation/ticketEscalation.sw.yaml

```diff
@@ -0,0 +1,115 @@
+specVersion: "0.8"
+id: ticketEscalation
+name: Ticket escalation
+annotations:
+  - "workflow-type/infrastructure"
+version: 0.0.1
+timeouts:
+  workflowExecTimeout:
+    duration: PT24H
+start: CreateTicket
+extensions:
+  - extensionid: workflow-uri-definitions
+    definitions:
+      notifications: "https://raw.githubusercontent.com/janus-idp/backstage-plugins/%40janus-idp/plugin-notifications-backend%401.0.3/plugins/notifications-backend/src/openapi.yaml"
+dataInputSchema:
+  failOnValidationErrors: true
+  schema: schemas/ticket-escalation-schema.json
+errors:
+  - name: timeoutError
+    code: TimedOut
+  - name: notAvailable
+    code: "404"
+functions:
+  - name: createNotification
+    operation: notifications#createNotification
+  - name: createK8sNamespace
+    operation: specs/kube.yaml#createCoreV1Namespace
+  - name: logInfo
+    type: custom
+    operation: "sysout:INFO"
+events:
+  - name: approvalEvent
+    source: ticket.listener
+    type: dev.parodos.escalation
+states:
+  - name: CreateTicket
+    type: operation
+    actions:
+      - subFlowRef: ticketingService
+        actionDataFilter:
+          toStateData: .createResponse
+    stateDataFilter:
+      input: '. += { request: {type: "create", namespace: .target.namespace, parentId: $WORKFLOW.instanceId} }'
+    transition: WaitForApprovalEvent
+  - name: GetTicket
+    type: operation
+    actions:
+      - subFlowRef: ticketingService
+        actionDataFilter:
+          toStateData: .getResponse
+    stateDataFilter:
+      input: '. += { request: { type: "get", ticketId: .createResponse.ticketId, ticket: .createResponse.ticket} }'
+    transition: CheckTicketState
+  - name: CheckTicketState
+    type: switch
+    dataConditions:
+      - condition: (.getResponse.status == "Approved")
+        transition:
+          nextState: CreateK8sNamespace
+    defaultCondition:
+      transition: WaitForApprovalEvent
+  - name: WaitForApprovalEvent
+    type: callback
+    action:
+      functionRef:
+        refName: logInfo
+        arguments:
+          message: "\"Waiting for approvalEvent: \\(.)\""
+    eventRef: approvalEvent
+    timeouts:
+      eventTimeout: PT10S
+      # This is not working for now, waiting for a fix to https://issues.redhat.com/browse/KOGITO-9811
```

Comment:
the issue is marked as resolved.

Reply:
Hard to test it using `kn-workflow`, will try again

Reply:
confirm it's been fixed. I will update it